src/frontend/Empty.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
```

Comment:
Using capital letters in file names like this ("Empty") causes me grief because I'm using mac, and running it on linux, and some developers will possibly be using windows, and because windows is not case sensitive and mac and linux are it causes no end of trouble down the line. Could you please rename the files from camel case to snake case, like this: Empty -> empty, StepNavigator -> step-navigator, etc...?

Reply:
Sure, but standard convention for components is to use capitals... I believe you even asked me to do this with classes?